app/containers/MessageBox/index.js

```diff
@@ -746,13 +848,62 @@ class MessageBox extends Component {
 					style={styles.mentionList}
 					data={mentions}
 					renderItem={({ item }) => this.renderMentionItem(item)}
-					keyExtractor={item => item._id || item.username || item}
+					keyExtractor={item => item._id || item.username || item.command || item}
 					keyboardShouldPersistTaps='always'
 				/>
 			</View>
 		);
 	};
 
+	renderCommandPreviewItem = (item) => {
```

Comment:
It should have the same horizontal space outside the slash icon
<img src="https://user-images.githubusercontent.com/804994/57856439-6f3b5c00-77c3-11e9-92ad-d4f1c770bed2.png" width="350" />

Reply:
couldn't understand it did you means 

> horizontal margin of slash icon should be equal?